docs/reference/components.cue

```diff
@@ -356,13 +356,13 @@ components: {
 			enabled: bool
 
 			if enabled {
-				adaptive_concurrency:       bool | *true
-				concurrency:                uint8 | *5
-				rate_limit_duration_secs:   uint8
-				rate_limit_num:             uint16
-				retry_initial_backoff_secs: uint8
-				retry_max_duration_secs:    uint8
-				timeout_secs:               uint8
+				adaptive_concurrency:       bool | *false
+				concurrency:                uint64
```

Comment:
Technically this is `usize` but I don't think there is a way to represent that in cue.